hydrofunctions/flood_stage.py

```diff
@@ -1,5 +1,3 @@
-from typing import Dict, List
```

Comment:
Ha! This is funny. I made the typing.py module before type hints were a thing. I will re-name the typing module in the future.

Reply:
Yeh it can happen. For reference: https://docs.python.org/3/library/typing.html

Reply:
I just renamed the typing module to prevent this conflict.